backend/core/src/auth/services/user.service.ts

```diff
@@ -130,6 +136,15 @@ export class UserService {
       delete dto.jurisdictions
     }
 
+    if (dto.newEmail && dto.appUrl) {
```

Comment:
I'm assuming this is coming from upstream, but I'm curious if you know why you need both the appUrl and the newEmail? Not sure I understand this part.

Reply:
The appUrl is needed for the confirmation email, and I think the newEmail is somehow encoded in that process so that it can be set once the user confirms the new email address.